src/windows/osd.c

```diff
@@ -431,20 +441,9 @@ int getifaddrs(struct ifaddrs **ifap)
 
 		for (ua = aa->FirstUnicastAddress; ua != NULL; ua = ua->Next) {
 			pSockAddr = ua->Address.lpSockaddr;
-			if (pSockAddr->sa_family != AF_INET)
-				continue;
-
-			subnet = 0;
-			mask = &subnet;
-			if (ConvertLengthToIpv4Mask(ua->OnLinkPrefixLength, mask) !=
-			    NO_ERROR) {
-				ret = -FI_ENODATA;
-				goto out;
-			}
-
 			fa = calloc(sizeof(*fa), 1);
 			if (!fa) {
-				ret -FI_ENOMEM;
+				ret = -FI_ENOMEM;
```

Comment:
Oops, how does the current code even compile?

Reply:
@shefty  It didn't compile locally, when I took it after merging. I wonder why..